WalletWasabi.Tests/UnitTests/WabiSabi/Models/MultipartyTransactionTests.cs

```diff
@@ -0,0 +1,327 @@
+using NBitcoin;
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using WalletWasabi.Helpers;
+using WalletWasabi.WabiSabi.Backend.Models;
+using WalletWasabi.WabiSabi.Models;
+using WalletWasabi.WabiSabi.Models.MultipartyTransaction;
+using WalletWasabi.Tests.Helpers;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.WabiSabi.Models
+{
+	public class MultipartyTransactionTests
+	{
+		private static MoneyRange DefaultAllowedAmounts = new (Money.Zero, Money.Coins(1));
```

Comment:
```suggestion
		private static MoneyRange DefaultAllowedAmounts = new(Money.Zero, Money.Coins(1));
```

It is typically written like this in WW AFAIK.

Reply:
I keep forgetting... I know Windows has CodeMaid which is used on the repository, but I'm using omnisharp in emacs, is there a tool to handle formatting?

Reply:
I'm not familiar with emacs. But in this case I would say, it's something one must remember. Visual Studio can format code but I'm not sure whether it gets everything right according to WW code style.